profiles/coreos/base/package.accept_keywords

```diff
@@ -27,8 +27,8 @@
 =net-misc/wget-1.21.2 ~amd64 ~arm64
 
 # Upgrade to GCC 10.3.0 to support latest glibc builds
-=sys-devel/binutils-2.37_p1 ~amd64 ~arm64
-=sys-libs/binutils-libs-2.37_p1 ~amd64 ~arm64
+=sys-devel/binutils-2.38-p2 ~amd64 ~arm64
+=sys-libs/binutils-libs-2.38-p2 ~amd64 ~arm64
 # This needs to be kept in-sync otherwise dev container contains
```

Comment:
Like @dongsupark says, please fixup lines 35-36 as well

Reply:
CI still shows warnings like:
```
10:44:17  --- Invalid atom in /mnt/host/source/src/third_party/coreos-overlay/profiles/coreos/base/package.accept_keywords: =sys-devel/binutils-2.38-p2
10:44:17  --- Invalid atom in /mnt/host/source/src/third_party/coreos-overlay/profiles/coreos/base/package.accept_keywords: =sys-libs/binutils-libs-2.38-p2
10:44:17  --- Invalid atom in /mnt/host/source/src/third_party/coreos-overlay/profiles/coreos/base/package.accept_keywords: =cross-x86_64-cros-linux-gnu/binutils-2.38-p2
10:44:17  --- Invalid atom in /mnt/host/source/src/third_party/coreos-overlay/profiles/coreos/base/package.accept_keywords: =cross-aarch64-cros-linux-gnu/binutils-2.38-p2
```

You would want to replace `-p2` with `-r2`.